### ISSUE TITLE
Write script to scrap data from older years.

### ISSUE THREAD

**Issue by panmari** [misch/gpCollect#9]

**Comment by panmari** [comment]
This is solved for years 2009-2012. Older versions still need to be added. But first we probably have to solve #11

**Comment by panmari** [comment]
Solved for years 2007 - 2008. What's left now is a parser for this funny format http://services.datasport.com/2006/lauf/gp/. I think it's best to use these pages http://services.datasport.com/2005/lauf/gp/ALFAA.HTM and go through every letter of the alphabet.

**Comment by panmari** [comment]
We got too many runners for 2006! (and some of the years before probably, too) According to http://services.datasport.com/2006/lauf/gp/RANG091-21.HTM there should be 10266, but we have

```
wc -l db/data/gp_bern_10m_*
    8557 db/data/gp_bern_10m_1999.csv
   28593 db/data/gp_bern_10m_2000.csv
   30726 db/data/gp_bern_10m_2001.csv
   31605 db/data/gp_bern_10m_2002.csv
   47784 db/data/gp_bern_10m_2003.csv
   50340 db/data/gp_bern_10m_2004.csv
   52796 db/data/gp_bern_10m_2005.csv
   56956 db/data/gp_bern_10m_2006.csv
   13600 db/data/gp_bern_10m_2007.csv
   14139 db/data/gp_bern_10m_2008.csv
   13350 db/data/gp_bern_10m_2009.csv
   15057 db/data/gp_bern_10m_2010.csv
   15681 db/data/gp_bern_10m_2011.csv
   15557 db/data/gp_bern_10m_2012.csv
   16232 db/data/gp_bern_10m_2013.csv
   17382 db/data/gp_bern_10m_2014.csv
   16262 db/data/gp_bern_10m_2015.csv
  444617 total
```

so almost 60k. No idea what happened <.<

**Comment by panmari** [comment]
Looks like every line appears multiple times <.<, waaaat?

**Comment by panmari** [comment]
Ok, I found the error, see 37806b27ceb3092b49c61288cb2fce35837eecfc

**Comment by panmari** [comment]
This is now done. Some final issues with data from 2007/2008 were solved in 7946265af72d4890409141e1d3b89779ecc34d7c
